src/v/test_utils/BUILD

```diff
@@ -37,6 +37,7 @@ redpanda_test_cc_library(
         "test_macros.h",
         "tmp_dir.h",
     ],
+    local_defines = ["IS_BTEST"],
```

Comment:
why is this needed for a header only library?

Reply:
clang-tidy